dspace/config/registries/schema-publicationIssue-types.xml

```diff
@@ -27,5 +27,149 @@
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>eissn</qualifier>
+        <scope_note>International Standard Serial Number (electronic Version)</scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>handle</qualifier>
+        <scope_note>Handle System Identifier</scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>igsn</qualifier>
+        <scope_note>International Geo Sample Number; a 9-digit alphanumeric code that uniquely identifies 
+            samples from our natural environment and related sampling features.
+        </scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>istc</qualifier>
+        <scope_note>International Standard Text Code; a unique “number” assigned to a textual work. An 
+            ISTC consists of 16 numbers and/or letters.
+        </scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>lissn</qualifier>
+        <scope_note>The linking ISSN or ISSN-L enables collocation or linking among different media 
+            versions of a continuing resource.
+        </scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>lsid</qualifier>
+        <scope_note>Life Science Identifiers; a unique identifier for data in the Life Science domain. 
+            Format: urn:lsid:authority:namespace:identifier:revision
+        </scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>pissn</qualifier>
+        <scope_note>International Standard Serial Number (print version)</scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>pmid</qualifier>
+        <scope_note>PubMed ID</scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>purl</qualifier>
+        <scope_note>Persistent Uniform Resource Locator</scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>upc</qualifier>
+        <scope_note>Universal Product Code is a barcode symbology used for tracking trade items in stores. 
+            Its most common form, the UPC-A, consists of 12 numerical digits.
+        </scope_note>
+    </dc-type>
+  
+    <dc-type>
+        <schema>creativework</schema>
+        <element>identifier</element>
+        <qualifier>url</qualifier>
+        <scope_note>Uniform Resource Locator</scope_note>
```

Comment:
Question for today's Entities meeting: Should this URL field (and URN below) be under `dc.identifier` alongside `dc.identifier.uri`?  Or would we rather these be under `creativework.identifier`?